File: responses/4T59SN-Z8Q/count_of_largest_integer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    // write your code in JavaScript
    if(A.lenght==0) return 0;
    var heightestNumber =-100000;
    var count=0;
    for(var i=0; i<A.length; i++){
        if(heightestNumber<A[i])
            heightestNumber = A[i];
    }
    for(var j=0; j<A.length; j++){
     if(A[j]==heightestNumber) count++;
    }

    return count;
}

File: responses/WFYFRM-9QS/count_of_largest_integer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # write your code in Python
    if A:
        cntDict = {}
        maxNum = A[0]
        count = 0
        for num in A:
            if num > maxNum:
                maxNum = num
                count = 1
            elif num == maxNum:
                count +=1
        return count
    else:
        return 0